recipe/recipe.yaml

```diff
@@ -0,0 +1,221 @@
+context:
+  version: 1.3.1
+  build_num: 2
+
+recipe:
+  name: zlib-split
+  version: ${{ version }}
+
+source:
+  url:
+    - http://zlib.net/zlib-${{ version }}.tar.gz
+    - https://gnupg.org/ftp/gcrypt/zlib/zlib-${{ version }}.tar.gz
+  sha256: 9a93b2b7dfdac77ceba5a558a580e74667dd6fede4585b91eefb60f03b72df23
+
+build:
+  number: ${{ build_num }}
+
+cache:
+  requirements:
+    build:
+      - cmake
+      - ninja
+      - if: cross_target_platform == "emscripten-wasm32"
+        then:
+          - emscripten
+        else:
+          - ${{ compiler('c') }}
+          - ${{ stdlib('c') }}
+  build:
+    script:
+      # rattler-build figures out the {.sh,.bat} extensions
+      file: build_global
+      # see https://github.com/prefix-dev/rattler-build/issues/1056
+      env:
+        CROSS_TARGET_PLATFORM: ${{ cross_target_platform }}
+
+outputs:
+  - package:
+      name: libzlib
+    build:
+      files:
+        - if: linux
+          then: lib/libz.so.*
+        - if: osx
+          then: lib/libz.*.dylib
+        - if: win
+          then:
+            - Library/bin/zlib.dll
+            - zlib.dll
+    requirements:
+      build:
+        - if: cross_target_platform == "emscripten-wasm32"
+          then:
+            - emscripten
+          else:
+            - ${{ compiler('c') }}
+            - ${{ stdlib('c') }}
+      host:
+      run:
+      run_exports:
+        - ${{ pin_subpackage('libzlib', upper_bound='x') }}
+      run_constraints:
+        - zlib ${{ version }} *_${{ build_num }}
+    tests:
+      - requirements:
+          build:
+            - if: cross_target_platform == "emscripten-wasm32"
+              then:
+                - nodejs
+        script:
+        - if: unix
+          then:
+            - test ! -f ${PREFIX}/lib/libz.a
+            - test ! -f ${PREFIX}/lib/libz${SHLIB_EXT}
+            - test ! -f ${PREFIX}/include/zlib.h
+          else:
+            - if not exist %LIBRARY_BIN%\zlib.dll exit 1
+            - if not exist %PREFIX%\zlib.dll exit 1
+
+  - package:
+      name: zlib
+    build:
+      files:
+        - if: linux
+          then: lib/libz.so
+        - if: osx
+          then: lib/libz.dylib
+        - if: unix
+          then:
+            - include/
+            - lib/pkgconfig/
+            - lib/libz.a
+          else:
+            - Library/include/
+            - Library/share/
+            - Library/lib/zlib.lib
+            - Library/lib/zdll.lib
+            - Library/lib/z.lib
+            - Library/lib/zlibstatic.lib
+            - Library/lib/pkgconfig/
+        - if: cross_target_platform == "emscripten-wasm32"
+          then:
+            - bin/libz.js
+            - bin/zlib.js
+    requirements:
+      build:
+        - if: cross_target_platform == "emscripten-wasm32"
```

Comment:
why do we need the compiler here if we don't do anything?

Reply:
This is an artefact of the conda-build recipe, where the run-exports from the global build don't get forwarded to the outputs, so they need(ed) another build-section to inherit the right REs for `libgcc` and `__glibc`.

That it's still here is an artefact of first the translation and me not noticing to remove it so far.